lib/basedriver/capabilities.js

```diff
@@ -148,10 +148,7 @@ function parseCaps (caps, constraints = {}, shouldValidateCaps = true) {
   // Check for non-prefixed, non-standard capabilities and log warnings if they are found
   let nonPrefixedCaps = findNonPrefixedCaps(caps);
   if (!_.isEmpty(nonPrefixedCaps)) {
-    log.warn(`The following capabilities are not standard capabilities and should have an extension prefix:`);
-    for (const cap of nonPrefixedCaps) {
-      log.warn(`  ${cap}`);
-    }
+    throw new errors.InvalidArgumentError(`All capabilities should have vendor prefix. The following capabilities did not have it: ${nonPrefixedCaps}`);
```

Comment:
`All non-standard capabilities should have a vendor prefix. The following capabilities did not have one:`